### PR TITLE
Visual show all objects on position

### PR DESCRIPTION
**Pull request by curusarn**
Solves: https://github.com/gdg-garage/space-tycoon/issues/75

Objects are discovered at the time of the click.
Order of shown objects:
- Clicked object
- Planets
- Ships
- Wrecks

### CODE REVIEW

File: web/js/bundle.js [hunk 1]
```diff
@@ -55,7 +55,13 @@ var ApiClient = /*#__PURE__*/function () {
...
      */
 
-    this.authentications = {};
+    this.authentications = {
+      'cookieAuth': {
+        type: 'apiKey',
+        'in': 'query',
+        name: 'SESSION_ID'
+      }
+    };
```
Comment (curusarn) [inline web/js/bundle.js:64]: Idk why this was generated

**Comment by malytomas** [comment]
why planets before ships?

File: web/js/bundle.js (hunk 1 above)
Comment (malytomas) [reply web/js/bundle.js:64]: changes to swagger were merged into master and the bundle.js was not updated yet. i have same changes in my branch.

**Comment by curusarn** [comment]
> why planets before ships?

There can be many ships but only one (or just a few) planets on the position. So it makes sense to show the planet before ships because it doesn't take up mych space and it is important. The planet not being shown because there is too many ships doesn't make sense to me. The opposite problem doesn't exist because there can't be a lot of planets on a single position.

File: web/js/bundle.js (hunk 1 above)
Comment (tivvit) [reply web/js/bundle.js:64]: #113

**Pull request merged**